sourcecode/hub/resources/views/user/my-account.blade.php

```diff
@@ -0,0 +1,18 @@
+<x-layout>
+    <x-slot:title>{{ trans('messages.my-account') }}</x-slot:title>
+
+    <h6>{{ trans('messages.change-profile-name') }}</h6>
+
+    <x-form action="{{ route('user.saveUsername') }}">
+        <x-form.field
```

Comment:
This must be populated with the existing name. So something like `:value="old('name', $user->name)"`. The current user must also be passed to the view in the controller.